geographer/downloaders/sentinel2_safe_unpacking.py

```diff
@@ -180,69 +214,71 @@ def safe_to_geotif_L2A(
         transform=out_default_reader.transform,
         dtype=out_default_reader.dtypes[0],
     ) as dst:
+        dst.nodata = nodata_val
+
+        with tqdm(total=count, desc=f"Extracting tif from {raster_name}.SAFE.") as pbar:
+
+            # write gml masks
+            for idx, (gml_name, gml_path) in enumerate(gml_mask_paths_dict.items()):
+                try:
+                    assert gml_path.is_file()
```

Comment:
One school of thought says assert should be used only for things you "know" to be true, i.e., only trigger an error if there's a bug in the code. I think here you want `if not is_file: raise ValueError`.

Reply:
Rewrote using FileNotFoundError